### PR TITLE
feat: tonymbugua-restencil

### PR DESCRIPTION
**Pull request by tonymbugua-outreach**
## Jira ID

[XX-XX]

### CODE REVIEW

File: yarn.lock [hunk 1]
```diff
@@ -433,15 +433,6 @@
   dependencies:
     "@octokit/openapi-types" "^16.0.0"
 
-"@outreach/semantic-release-circleci-orb@^1.1.9":
```
Comment (chatchai-outreach) [inline yarn.lock:436]: release failed due to missing **semantic-release-circleci-orb** - let's add them back

File: package.json [hunk 2]
```diff
@@ -1,7 +1,6 @@
 {
   "license": "UNLICENSED",
   "devDependencies": {
-    "@outreach/semantic-release-circleci-orb": "^1.1.9",
```
Comment (chatchai-outreach) [inline package.json:4]: release failed due to missing semantic-release-circleci-orb - let's add them back

File: yarn.lock (hunk 1 above)
Comment (tonymbugua-outreach) [reply yarn.lock]: added back

File: yarn.lock [hunk 3]
```diff
@@ -442,6 +442,15 @@
     aggregate-error "^3.1.0"
     execa "^5.1.0"
 
+"@outreach/semantic-release-circleci-orb@^1.1.9":
```
Comment (chatchai-outreach) [inline yarn.lock:445]: dup?

**Review by chatchai-outreach — APPROVED** [review]

**Pull request merged**